documentation/book/oauth/proc-oauth-broker-config.adoc

```diff
@@ -0,0 +1,109 @@
+* {oauth} authorization server is deployed
+
+.Procedure
+
+. Update the Kafka broker configuration (`Kafka.spec.kafka`) of your `kafka` resource in an editor.
++
+[source,shell]
+----
+kubectl edit kafka my-cluster
+----
+
+. Configure the Kafka broker listener configuration
++
+The configuration for each listener does not have to be the same, as they are independent.
++
+For example:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+listeners:
+ plain:
+   authentication:
+     type: oauth <1>
+     validIssuerUri: _https://<authorization-server-address>/auth/realms/demo_ <2>
+     jwksEndpointUri: _https://<authorization-server-address>/auth/realms/demo/protocol/openid-connect/certs_ <3>
+     userNameClaim: preferred_username <4>
+ tls:
+   authentication:
+     type: oauth
+     validIssuerUri: _https://<authorization-server-address>/auth/realms/demo_
+     jwksEndpointUri: _https://<authorization-server-address>/auth/realms/demo/protocol/openid-connect/certs_
+     userNameClaim: preferred_username
+ external:
+   type: loadbalancer
+
+   //Option 1
+   authentication: <5>
+     type: oauth
+     validIssuerUri: _https://<authorization-server-address>/auth/realms/external_
+     jwksEndpointUri: _https://<authorization-server-address>/auth/realms/external/protocol/openid-connect/certs_
+     userNameClaim: preferred_username
+     tlsTrustedCertificates: <6>
+     - secretName: oauth-server-cert
+       certificate: ca.crt
+   disableTlsHostnameVerification: true <7>
+
+   //Option 2
+   authentication: <8>
+     type: oauth
+     validIssuerUri: _https://<authorization-server-address>/auth/realms/external_
+     introspectionEndpointUri: _https://<authorization-server-address>/auth/realms/external/protocol/openid-connect/token/introspect_
+     clientId: kafka-broker
+     clientSecret:
+       secretName: my-cluster-oauth
+       key: clientSecret
+
+----
+<1> Listener type set to `oauth`.
+<2> URI of the token issuer used for authentication.
+<3> URI of the JWKS certificate endpoint used for local JWT validation.
+<4> The user name profile claim (or key) that contains the actual user name in the token. The user name is the _principal_ used to identify the user. The `userNameClaim` value will depend on the authentication flow and the authorization server used.
+<5> OPTION 1: TLS connection to the authorization server.
+<6> (Optional) Trusted certificates for TLS connection to the authorization server.
+<7> Enable TLS hostname verification. Default is `false`.
+<8> OPTION 2: Introspection endpoint to connect directly to the authorization server.
+
+. Save and exit the editor, then wait for rolling updates to complete.
+
+. Check the update in the logs or by watching the pod state transitions:
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl logs -f ${POD_NAME} -c ${CONTAINER_NAME}
+kubectl get po -w
+----
++
+The rolling updates configure the brokers to send messages using {oauth} authentication.
```

Comment:
The brokers do more than send messages. Also, there should be just one rolling update. So maybe this would be better:

```suggestion
The rolling update configures the brokers to use {oauth} authentication.
```